docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -1,115 +1,106 @@
 # Importing an existing volume
 
-Use this information to import volumes created externally from the IBM® block storage CSI driver by using a persistent volume (PV) yaml file.
+Use this information to import volumes created externally from the IBM® block storage CSI driver by using a persistent volume (PV) YAML file.
 
-Before starting to import an existing volume, find the following information in the existing volume in order to include the information in the persistent volume (PV) yaml file:
+Before starting to import an existing volume, find the following information in the existing volume in order to include the information in the persistent volume (PV) YAML file:
+- `volumeHandle`
+- `volumeAttributes` (optional)
+  
+  Including:
 
--   `volumeHandle`
--   `volumeAttributes` (optional)
-
-    Including:
-
-    -   `pool_name`: _Name of Pool where volume is located_ (Listed as `pool_id` for DS8000® Family systems.)
-    -   `storage_type`: <SVC | A9K | DS8K>
-    -   `volume_name`: _Volume name_
-    -   `array_address`: _Array address_
+    - `pool_name`: _Name of Pool where volume is located_ (Listed as `pool_id` for DS8000® Family systems.)
+    - `storage_type`: <`SVC` | `A9K` | `DS8K`>
+    - `volume_name`: _Volume name_
+    - `array_address`: _Array address_
 
 To find the `volumeHandle`, use one of the following procedures:
 
-- Through command line (for Spectrum Virtualize Family):
 
-  `lsvdisk <volume name> | grep vdisk_UID`
-    
-  ```
-  lsvdisk vol0 | grep vdisk_UID
-  vdisk_UID 600507640082000B08000000000004FF
-  ```
+- **Through command line (for Spectrum Virtualize Family):**
 
-- Through command line (for FlashSystem A9000 and A9000R):
+  Find the `vdisk_UID` parameter, by using the `lsvdisk` command.
```

Comment:
note that, in addition to `vdisk_UID`, the `id` is now required as well (for replication).
in addition, I think svc [use](https://www.ibm.com/docs/en/flashsystem-900/1.6.1?topic=commands-lsvdisk) the term `attribute` (and not `parameter`)

Reply:
fixed - DONE
i'm not adding in information now for replication for A9K. we'll open a ticket and update when needed.

Reply:
as I wrote, I think it is needed now, so that users won't have to re-do this in the future just to enable replication (it's the longest and hardest procedure we have, so I imagine it would be very frustrating from them to re-do it).

Reply:
in addition, I would change:
`When using the remote copy function find the `ID` attribute. This is also found by using the `lsvdisk` command.`
to something like:
`In order to allow the remote copy function, find the `id` attribute. This is also found by using the `lsvdisk` command.`

Reply:
or just require the user to always find the `id` as well, in case it will be needed for other features as well in the future

Reply:
ID is listed as such (with capitals) in the [SV documentation](https://www.ibm.com/docs/en/flashsystem-9x00/8.4.x?topic=commands-lsvdisk)
![image](https://user-images.githubusercontent.com/54570654/135214394-ff81f6a1-bef9-41fc-a5e1-8749c5aa22e6.png)

if you want them to be able to find it for also A9K and DS8K i need to know what they're looking for there, including the GUI.

Reply:
@roysahar-ibm  what property do they use in the SV GUI for "ID"?

Reply:
@oriyarde  I updated the wording of finding the ID attribute (per [your comment above](https://github.com/IBM/ibm-block-csi-driver/pull/360#discussion_r716519765)).

Reply:
DONE FYI @ArbelNathan  @oriyarde @roysahar